types/shopify-buy/index.d.ts

```diff
@@ -281,6 +284,13 @@ declare namespace ShopifyBuy {
         value: string;
     }
 
+    export interface CustomAttributeV2 {
+        customAttributes: {
+            key: string | undefined;
```

Comment:
Would it better to do `key?: string`?

Reply:
Yes, it will be better.

Reply:
Can you reference some documentation that indicates that custom attributes can have optional keys?